packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -93,6 +93,10 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
                     xyTree: treeResponse.model.entries,
                     columns
                 });
+            } else {
+                this.setState({
+                    outputStatus: treeResponse.status
+                });
             }
             return treeResponse.status;
```

Comment:
if status is FAILED, then the XY chart is not updated and the "Analysis Running" message is shown. It can be reproduced with the apt trace in the tracevizlab when opening the first time the event matching xy chart. The failed case needs to be updated as well.

Reply:
Thank you for figuring out this bug! Should these additional changes be a separate PR or just a separate commit in this PR (with an updated description)?

Reply:
@ebugden  I'm ok to track it in a separate issue. This PR alone is already an good improvement and it's good to get it merged. Could you please open a separate issue tracker?

Reply:
For those trying to reproduce the failed status case with the Event Matching Latency analysis: **Event Matching Latency only returns status FAILED the first time the analysis is opened on server (0:26-0:30)**, otherwise it returns the status COMPLETED (0:37-0:40). 

https://user-images.githubusercontent.com/28311615/130796784-624f5c6d-48d8-4cb6-9a9b-4703d439c61f.mp4

(Forgive my debugging via printing, I don't know how to debug using gitpod.)